.github/actions/check_version/action.yml

```diff
@@ -34,5 +34,9 @@ runs:
         grep "^version = \"$oso_version\"$" polar-core/Cargo.toml
         grep "^version = \"$oso_version\"$" polar-c-api/Cargo.toml
         grep "^version = \"$oso_version\"$" polar-wasm-api/Cargo.toml
+      shell: bash
+    - name: Check VSCode extension version matches
+      run: |
         grep "^version = \"$oso_version\"$" polar-language-server/Cargo.toml
+        grep "\"version\": \"$oso_version\"" vscode/oso/package.json
```

Comment:
This'll keep the extension version in line with the core version until we come up with a more flexible versioning scheme for the extension and an automated deployment flow that enforces it.